presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java

```diff
@@ -213,6 +214,15 @@ public static Block createLongRepeatBlock(int value, int length)
         return builder.build();
     }
 
+    public static Block createTimestampsWithTimezoneBlock(Long... values)
```

Comment:
BlockAssertions already contains a method createTimestampSequenceBlock, should I get rid of that as well?